composer.json

```diff
@@ -25,7 +25,7 @@
     "require": {
         "php": "^8.1",
         "silverstripe/admin": "^2",
-        "silverstripe/framework": "^5.3",
+        "silverstripe/framework": "^5.4",
```

Comment:
No need to bump - the config will just be unused for earlier versions.

Reply:
Updated